lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -918,110 +941,44 @@ def install_puppetlabs_dev_repo ( host, package_name, build_version,
           # url type
           _, protocol, hostname = opts[:dev_builds_url].partition /.*:\/\//
           dev_builds_url = protocol + hostname
+          dev_builds_url = opts[:dev_builds_url] if variant =~ /^(fedora|el|centos)$/
```

Comment:
overwriting, as this is something that was in the case statement before I refactored it out. I've refactored it here so that everyone can just use `dev_builds_url`, rather than either `dev_builds_url` or `opts[:dev_builds_url]`, as they had been before